File: examples/db/renameTable/node/renameTable.js
const { DbService } = require("m3o/db");

// Rename a table
async function renameTable() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.renameTable({
    from: "events",
    to: "events_backup",
  });
  console.log(rsp);
}

renameTable();

File: examples/db/renameTable/go/renameTable.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// Rename a table
func RenameTable() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.RenameTable(&db.RenameTableRequest{
		From: "events",
		To:   "events_backup",
	})
	fmt.Println(rsp, err)
}
